sigma/pipelines/panther/sdyaml_transformation.py

```diff
@@ -53,7 +53,10 @@ def _detect_log_types(self, rule: SigmaRule) -> [str]:
             log_types.append("AWS.CloudTrail")
 
         cli_context = click.get_current_context(silent=True)
-        if cli_context and "crowdstrike_fdr" in cli_context.params["pipeline"]:
-            log_types.append("Crowdstrike.FDREvent")
+        if cli_context:
+            if "crowdstrike_fdr" in cli_context.params["pipeline"]:
+                log_types.append("Crowdstrike.FDREvent")
 
+            if "carbon_black_panther" in cli_context.params["pipeline"]:
+                log_types.append("CarbonBlack.EndpointEvent")
```

Comment:
Will we support the other Carbon Black log types eventually?

As of right now, PAT supports `EndpointEvent` and three additional log types:
- `CarbonBlack.AlertV2`
- `CarbonBlack.Audit`
- `CarbonBlack.EndpointEvent`
- `CarbonBlack.WatchlistHit`

Reply:
cc @arielkr256

Reply:
It's not a blocker for this PR but I was curious. 😅

Reply:
This is correct, Sigma rules will only look at `CarbonBlack.EndpointEvent`